containers/cell-type-annotations-to-factors/context/main.py

```diff
@@ -0,0 +1,64 @@
+import argparse
```

Comment:
I'm not sure how this PR relates to the other one I reviewed, but the comments there apply.